src/foundry/common/constants.d.mts

```diff
@@ -944,21 +1448,163 @@ export declare const FONT_WEIGHTS: {
+  img: ["height", "src", "width", "usemap", "sizes", "srcset", "alt"];
+  input: [
+    "checked",
+    "disabled",
+    "name",
+    "value",
+    "placeholder",
+    "type",
+    "alt",
+    "height",
+    "list",
+    "max",
+    "min",
+    "placeholder",
+    "readonly",
+    "size",
+    "src",
+    "step",
+    "width",
+  ];
+  label: ["for"];
+  li: ["value"];
+  map: ["name"];
+  meter: ["value", "min", "max", "low", "high", "optimum"];
+  ol: ["reversed", "start", "type"];
+  optgroup: ["disabled", "label"];
+  option: ["disabled", "selected", "label", "value"];
+  progress: ["max", "value"];
+  select: ["name", "disabled", "multiple", "size"];
+  source: ["media", "sizes", "src", "srcset", "type"];
+  table: ["border"];
+  td: ["colspan", "headers", "rowspan"];
+  textarea: ["rows", "cols", "disabled", "name", "readonly", "wrap"];
+  time: ["datetime"];
+  th: ["abbr", "colspan", "headers", "rowspan", "scope", "sorted"];
+  track: ["default", "kind", "label", "src", "srclang"];
+  video: ["controls", "height", "width", "loop", "muted", "poster", "src", "autoplay"];
+}>;
+export type ALLOWED_HTML_ATTRIBUTES = ValueOf<typeof ALLOWED_HTML_ATTRIBUTES>;
+
+/**
+ * The list of trusted iframe domains.
+ */
+export declare const TRUSTED_IFRAME_DOMAINS: ["google.com", "youtube.com"];
+export type TRUSTED_IFRAME_DOMAINS = ValueOf<typeof TRUSTED_IFRAME_DOMAINS>;
+
+/**
+ * Available themes for the world join page.
+ */
+export const WORLD_JOIN_THEMES: Readonly<{
+  default: "WORLD.JoinThemeDefault";
+  minimal: "WORLD.JoinThemeMinimal";
+}>;
+export type WORLD_JOIN_THEMES = ValueOf<typeof WORLD_JOIN_THEMES>;
+
+/**
+ * Setup page package progress protocol.
+ */
+export declare const SETUP_PACKAGE_PROGRESS: Readonly<{
+  ACTIONS: {
+    CREATE_BACKUP: "createBackup";
+    RESTORE_BACKUP: "restoreBackup";
+    DELETE_BACKUP: "deleteBackup";
+    CREATE_SNAPSHOT: "createSnapshot";
+    RESTORE_SNAPSHOT: "restoreSnapshot";
+    DELETE_SNAPSHOT: "deleteSnapshot";
+    INSTALL_PKG: "installPackage";
+    LAUNCH_WORLD: "launchWorld";
+    UPDATE_CORE: "updateCore";
+  };
+
+  STEPS: {
+    ARCHIVE: "archive";
+    CHECK_DISK_SPACE: "checkDiskSpace";
+    CONNECT_WORLD: "connectWorld";
+    CONNECT_PKG: "connectPackage";
+    MIGRATE_CORE: "migrateCore";
+    MIGRATE_SYSTEM: "migrateSystem";
+    DOWNLOAD: "download";
+    EXTRACT: "extract";
+    INSTALL: "install";
+    CLEANUP: "cleanup";
+    COMPLETE: "complete";
+    DELETE: "delete";
+    ERROR: "error";
+    VEND: "vend";
+    SNAPSHOT_MODULES: "snapshotModules";
+    SNAPSHOT_SYSTEMS: "snapshotSystems";
+    SNAPSHOT_WORLDS: "snapshotWorlds";
+  };
+}>;
+export namespace SETUP_PACKAGE_PROGRESS {
+  type ACTIONS = ValueOf<typeof SETUP_PACKAGE_PROGRESS.ACTIONS>; // eslint-disable-line @typescript-eslint/no-unused-vars
+  type STEPS = ValueOf<typeof SETUP_PACKAGE_PROGRESS.STEPS>; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Don't these need an `export` as well? That would likely also make the eslint-ignores not needed.

Reply:
Uh. For some reason, I had a compilation early in this when exporting and thus ruled it out. I must have done something wrong early and didn't check again. I'll correct it.